docs/user/openstack/bind-vfio-pci-kernel-module.md

```diff
@@ -0,0 +1,144 @@
+# Binding the vfio-pci kernel module to vhostuser port.
+
+When a VM is attached to a port of type dpdkvhostuserclient, as is the case when vms are
+spun up inside an OpenStack deployment configured for [ovs-dpdk][1] it is possible to use a pmd driver like the [VFIO
```

Comment:
Link to the PMD docs would be helpful for those less well versed in all things DPDK. [This doc](https://doc.dpdk.org/guides/prog_guide/poll_mode_drv.html) looks as good as it gets?

Also, `s/\<pmd\>/PMD/g`